environments/production/hiera/common.json

```diff
@@ -47,7 +47,7 @@
 
     "web_domain": "unity.ac",
     "web_domains_external": ["*.oaeproject.org", "collab.lib.cam.ac.uk", "oae.gatech.edu", "oae.its.txstate.edu", "shib-sp.unity.ac", "open.vu.nl"],
-    "web_domains_redirect": ["research"],
+    "web_domains_redirect": ["research", "bath", "britishcouncil", "gla", "kingston", "plymouth", "sun", "sussex"],
```

Comment:
@davidoae : Is *Research taking care of the communication with all of these places to inform them about the URL change of their tenancy?